python_modules/dagster-airflow/dagster_airflow_tests/conftest.py

```diff
@@ -67,17 +91,123 @@ def plugins_path(airflow_home):
 @pytest.fixture(scope='module')
 def host_tmp_dir():
     mkdir_p('/tmp/results')
+    return '/tmp/results'
 
 
 @pytest.fixture(scope='module')
-def airflow_test(airflow_home, docker_client, dags_path, plugins_path, host_tmp_dir):
+def airflow_test(docker_image, dags_path, plugins_path, host_tmp_dir):
+
+    assert docker_image
 
     plugin_definition_filename = 'dagster_plugin.py'
+
+    plugin_path = os.path.abspath(os.path.join(plugins_path, plugin_definition_filename))
+
+    temporary_plugin_path = None
+
+    try:
+        if os.path.exists(plugin_path):
+            temporary_plugin_file = tempfile.NamedTemporaryFile(delete=False)
+            temporary_plugin_file.close()
+            temporary_plugin_path = temporary_plugin_file.name
+            shutil.copyfile(plugin_path, temporary_plugin_path)
+
+        shutil.copyfile(
+            script_relative_path(os.path.join('..', 'dagster_airflow', plugin_definition_filename)),
+            plugin_path,
+        )
+
+        mkdir_p(os.path.abspath(dags_path))
+        sys.path.append(os.path.abspath(dags_path))
+
+        created_init_py = False
+        init_py_path = os.path.join(os.path.abspath(dags_path), '__init__.py')
+        if not os.path.exists(init_py_path):
+            with open(init_py_path, 'a'):
+                pass
+            created_init_py = True
+
+        subprocess.check_output(['airflow', 'initdb'])
+
+        # necromancy; follows airflow.operators.__init__
+        reload_module(airflow.plugins_manager)
+        for operators_module in airflow.plugins_manager.operators_modules:
+            sys.modules[operators_module.__name__] = operators_module
+            globals()[operators_module._name] = operators_module
+
+        from airflow.operators.dagster_plugin import DagsterOperator
+
+        del DagsterOperator
```

Comment:
what is happening here

Reply:
just testing that it's in fact importable